### PR TITLE
New Slash docs homepage

### PR DESCRIPTION
**Pull request by bucanero**
This PR adds a new Slash docs homepage, following the new design

Fixes GQLSAAS-810

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -5,19 +5,77 @@ weight = 4
...
+      Slash GraphQL provides a <code>/graphql</code> Backend for your app.
+    </p>
+    <img class="hero-deco" src="/images/hero-deco.png" />
+  </div>
+  <div class="item">
+    <div class="icon"><i class="lni lni-play" aria-hidden="true"></i></div>
+    <a  href="{{< relref "slash-quick-start.md">}}">
+      <h2>Slash Quick Start</h2>
+      <p>
+        Get started with a Slash GraphQL Schema
```
Comment (aaroncarey) [inline content/_index.md:21]:
```suggestion
        Get started with a simple "to-do" list app built on Slash GraphQL
```

File: content/_index.md [hunk 2]
```diff
@@ -5,19 +5,77 @@ weight = 4
...
+      </p>
+    </a>
+  </div>
+  <div class="item">
+    <div class="icon"><i class="fa fa-university" aria-hidden="true"></i></div>
+    <a href="{{< relref "advanced-queries.md">}}">
+      <h2>Advanced Queries with DQL</h2>
+      <p>
+        How to interact with your database via the gRPC endpoint
```
Comment (aaroncarey) [inline content/_index.md:30]:
```suggestion
        Generate advanced queries to interact with your database using the HTTP or gRPC endpoint 
```

File: content/_index.md [hunk 3]
```diff
@@ -5,19 +5,77 @@ weight = 4
...
+      </p>
+    </a>
+  </div>
+  <div class="item">
+    <div class="icon"><i class="fa fa-code-fork" aria-hidden="true"></i></div>
+    <a href="{{< relref "one-click-deploy.md">}}">
+      <h2>One-click Deploy</h2>
+      <p>
+        How to deploy sample apps in a fresh backend instance
```
Comment (aaroncarey) [inline content/_index.md:39]:
```suggestion
        Deploy sample apps in minutes from a fresh backend instance
```

File: content/_index.md [hunk 4]
```diff
@@ -5,19 +5,77 @@ weight = 4
...
+    </a>
+  </div>
 
-Please see the following topics:
+  <div class="item">
+    <div class="icon"><i class="lni lni-graduation" aria-hidden="true"></i></div>
+    <a href="{{< relref "security.md">}}">
+      <h2>Securing your GraphQL endpoint</h2>
+      <p>
+        Tips for securing your Slash GraphQL Backend
```
Comment (aaroncarey) [inline content/_index.md:49]:
```suggestion
        Tips for securing your Slash GraphQL backend
```

File: content/_index.md [hunk 5]
```diff
@@ -5,19 +5,77 @@ weight = 4
...
+      </p>
+    </a>
+  </div>
+  <div class="item">
+    <div class="icon"><i class="lni lni-fireworks" aria-hidden="true"></i></div>
+    <a href="{{< relref "migrating-from-hosted-dgraph.md">}}">
+      <h2>Migrating from hosted Dgraph</h2>
+      <p>
+        Easily migrate your existing Dgraph app over to Slash
```
Comment (aaroncarey) [inline content/_index.md:58]:
```suggestion
        Easily migrate your existing Dgraph app to Slash
```

File: content/_index.md [hunk 6]
```diff
@@ -5,19 +5,77 @@ weight = 4
...
+      </p>
+    </a>
+  </div>
+  <div class="item">
+    <div class="icon"><i class="fa fa-cog" aria-hidden="true"></i></div>
+    <a href="{{< relref "admin/_index.md">}}">
+      <h2>Administering your Backend</h2>
+      <p>
+        How to programatically set your schema, and import/export your data
```
Comment (aaroncarey) [inline content/_index.md:67]:
```suggestion
        Programmatically set and update your schema, import or export your data, and switch or clone your Slash backend
```

**Review by aaroncarey — COMMENTED** [review]
What a nice looking homepage! I suggested a few changes to card text. Thanks for the chance to review!

**Review by mbj36 — APPROVED** [review]

**Comment by mbj36** [comment]
Great Job @bucanero

**Pull request merged**